calibration/DOFSTK/CGINS-DOFSTK-02499__20200718.csv

```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2499,CC_frequency_offset,-8.356636e+002,source_file DOFST-K_SBE43F_SN_2499_Calibration_2020-07-18.zip > 2499.cal and SBE 43F O2499 SOCAdjusted 30Jul20.pdf
+43-2499,CC_oxygen_signal_slope,3.4847e-004,use adjusted Soc
```

Comment:
All values look good.  Need to be sure this slope ('adjusted Soc') value is in the instrument.